src/functions/template-tags/organizer.php

```diff
@@ -148,9 +148,18 @@ function tribe_has_organizer( $postId = null ) {
 	 */
 	function tribe_get_organizer_email( $postId = null ) {
 		$postId = Tribe__Events__Main::postIdHelper( $postId );
-		$output = esc_html( tribe_get_event_meta( tribe_get_organizer_id( $postId ), '_OrganizerEmail', true ) );
+		$email  = esc_html( tribe_get_event_meta( tribe_get_organizer_id( $postId ), '_OrganizerEmail', true ) );
 
-		return apply_filters( 'tribe_get_organizer_email', $output );
+		/**
+		 * Allows for the organizer email to be filtered.
+		 *
+		 * Please note that obfuscation of email is done in subsequent line using the `antispambot` function.
+		 *
+		 * @param string $email The organizer email.
+		 */
+		$filtered_email = apply_filters( 'tribe_get_organizer_email', $email );
```

Comment:
Might it be better to either to use antispambot() itself as a `tribe_get_organizer_email` filter (so it can be unhooked) or else use it before the apply_filters() call and pass the original and unmodified email address as a 2nd param?